CDN/postpreinstall.sh

```diff
@@ -47,7 +47,7 @@ systemctl restart appserver
 systemctl enable devmgmt
 systemctl restart devmgmt
 
-reboot
+reboot_device
```

Comment:
Can you explain me the rational of changing this ?

Reply:
Since, the name of the routine was reboot. `reboot now` was causing the routine to get called recursively and not really triggering a reboot. Thus, the rename.